### ISSUE TITLE
Remove participation for Quest

### ISSUE THREAD

**Issue by Kalinow** [viadee/sonarQuest#264]
Hey,

I think it would be useful to add the function "remove Participation" for quest. 
(Like in "view-participated-quest-component")

Maybe we can extend the function: If the Quest-Participant removes a task ( and he has no more participation in Quest) then he should loose the quest participation.

**Comment by Christoph-Meyer** [comment]
Good idea actually - I added some labels because this seems to be not too difficult to code.
